src/indexing/EventIndex.js

```diff
@@ -0,0 +1,412 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import PlatformPeg from "../PlatformPeg";
+import MatrixClientPeg from "../MatrixClientPeg";
+
+/**
```

Comment:
Ah, it's tagging the property functions at this class level... I think the JSDoc lint is a bit confused, but maybe you can workaround it by making this not a JSDoc comment by changing this to a single star here...?